Sources/Clappr/Classes/Plugin/Core/UICorePlugin.swift

```diff
@@ -9,6 +9,16 @@ open class UICorePlugin: SimpleCorePlugin, UIPlugin {
         }
     }
     
+    #if os(tvOS)
+    func updateFocus(userInfo: EventUserInfo) {
```

Comment:
Is this functionality applicable to all types of `UIView`s / `UICorePlugin`s ?
Current usage looks like a specific kind of `Plugin` only functionality.

About the name, the intent is not to actually update the focus but rather to request focus for itself.

Reply:
We changed it to 2 functions, 1 to request focus and other to request a update on focus engine.